metrics/api/src/main/java/io/helidon/metrics/api/SystemTagsManagerImpl.java

```diff
@@ -17,9 +17,7 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Collections;
 import java.util.HashSet;
-import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
```

Comment:
the static field `METRICS_CONFIG_OVERRIDES` is unused as well as the method `scopeIterable`.